config/content.d/developer.rackspace.com.json

```diff
@@ -33,8 +33,7 @@
           "/docs/private-cloud/rpc/v14/": "https://github.com/rackerlabs/docs-rpc/v14/",
           "/docs/private-cloud/rpc/v13/": "https://github.com/rackerlabs/docs-rpc/v13/",
           "/docs/private-cloud/rpc/v12/": "https://github.com/rackerlabs/docs-rpc/v12/",
-          "/docs/private-cloud/release-notes/v14": "https://github.com/rcbops/rpc-openstack/v14/",
-          "/docs/private-cloud/release-notes/v13": "https://github.com/rcbops/rpc-openstack/v13/",
+          "/docs/private-cloud/release-notes/source/": "https://github.com/rcbops/rpc-openstack/tree/master/releasenotes/source",
```

Comment:
So the first bit (the key in the key:value pair) is the actual URL ending. It gets added to the end of `https://developer.rackspace.com`. Do you want the URL to be `https://developer.rackspace.com/docs/private-cloud/release-notes/source/`?
For the last bit, you want to take the `contentIDBase` found in the `_deconst.json` file [here](https://github.com/rcbops/rpc-openstack/blob/master/releasenotes/source/_deconst.json#L2). Think of this as a mapping system that looks for a `_deconst.json` file and assigns a unique URL to the directory where the correct `contentIDBase` lives.
Here's the [relevant bits of the deconst docs](http://deconst-docs-rtd.readthedocs.io/en/latest/writing-docs/author/index.html#adding-a-new-content-repository) for this stuff.